File: PPPK_Java/src/hr/algebra/model/Vozilo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hr.algebra.model;

/**
 *
 * @author Marko
 */
public class Vozilo {
    
    private int idVozilo;
    private String marka;
    private String tip;
    private String godinaProizvodnje;
    private String inicijalnoStanjeKilometara;

    public Vozilo(String marka, String tip, String godinaProizvodnje, String inicijalnoStanjeKilometara) {
        this.marka = marka;
        this.tip = tip;
        this.godinaProizvodnje = godinaProizvodnje;
        this.inicijalnoStanjeKilometara = inicijalnoStanjeKilometara;
    }

    public Vozilo(int idVozilo, String marka, String tip, String godinaProizvodnje, String inicijalnoStanjeKilometara) {
        this(marka, tip, godinaProizvodnje, inicijalnoStanjeKilometara);
        this.idVozilo = idVozilo;
    }

    public int getIdVozilo() {
        return idVozilo;
    }

    public String getMarka() {
        return marka;
    }

    public String getTip() {
        return tip;
    }

    public String getGodinaProizvodnje() {
        return godinaProizvodnje;
    }

    public String getInicijalnoStanjeKilometara() {
        return inicijalnoStanjeKilometara;
    }

    @Override
    public String toString() {
        return String.format("%s %s", marka, tip);
    }
    
}

File: PPPK_Web/PPPK_Web/Models/Vozilo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PPPK_Web.Models
{
    public class Vozilo
    {
        public int IDVozilo { get; set; }
        public string Marka { get; set; }
        public string Tip { get; set; }
        public string GodinaProizvodnje { get; set; }
        public string InicijalnoStanjeKilometara { get; set; }
        public string Naziv { get => Marka + " " + Tip; }

        public Vozilo(int iDVozilo, string marka, string tip, string godinaProizvodnje, string inicijalnoStanjeKilometara)
        {
            IDVozilo = iDVozilo;
            Marka = marka;
            Tip = tip;
            GodinaProizvodnje = godinaProizvodnje;
            InicijalnoStanjeKilometara = inicijalnoStanjeKilometara;
        }
        public Vozilo()
        {
        }
    }
}